src/containers/paper-canvas.jsx

```diff
@@ -266,10 +322,9 @@ class PaperCanvas extends React.Component {
         return (
             <canvas
                 className={styles.paperCanvas}
-                height="360px"
                 ref={this.setCanvas}
                 style={{cursor: this.props.cursor}}
-                width="480px"
+                resize="true"
```

Comment:
Is `resize` a React/Paper-specific attribute?

Reply:
yes https://github.com/paperjs/paper.js/blob/a61e83edf2ed1870bd41bad135f4f6fc85b0f628/examples/css/style.css#L9

Reply:
That looks like example CSS--is it actually included in this repo/imported from Paper somehow? I can't find any equivalent CSS here.

Reply:
Whoops! Here we go https://github.com/paperjs/paper.js/blob/9de1d4a2f36a2a4c62fbf71675bd92fb2b921460/src/view/View.js#L63